### DISCUSSION TITLE
授权码过期了，麻烦大佬帮忙激活一下 感谢

### DISCUSSION THREAD

**Discussion by lsq6789** [LFTEC/LearningFucker#54] [General]
请将下面代码复制到Github讨论区中，收到激活码后，调用activate命令进行激活
------------------------------License------------------------------
RaAshgXrLmcw9+DpYVmmSnoSKDLqj0g4ZCYqBudgu9uQ/ZpPwDX9LWm9RSu38sd3L5t
TU7YKZHQ1dPKmRlZvXTEy7grdS+tZi3j2UmUOUI4gNaqBeu4Abure8xIH/N6T71VsOI
Ev9IxSX//XvqJYoLAdQhSIuT5GgVHY9TzkCz6gL7Iw6YAdgCgH2VqHnAAmPSW4Lpvxj
E2zvqw2Jxam8TwoA8kGV1fvn/aLO49UPviV+fVqJBMii6Bt08Xpk6pqS9qh/3XSiSH4
x8hQXEADoeJR5XMiz0dlBKPBND6vRs2LLKYInRDrb/WZO3uk8Rt0v8DiYRKVTh0CqYc
aGkgkpDnmtp6eGFlAE9ok0I3mn/GaW6pZpV6Ve7lB4Jskk/mG1UWelkBJ04UW3xJHKL
SO1P4ReYUpUzJyH5oGBRgYy/L/IF+3mJkuianvIFr7s2CViJneFLoRdZgMT6EVjaSjj
lJEj2Vq1N+rMlh/TGC23RFAuJHBRJkq3cAEB8LZitkp29/EAeIl5xjrGiW2hRNdGwJZ
hEK0vJKFPsdHTBcg4zPL0XKEszkyhAJDdeK6+7FhcMG70spsAsG5OVxDSt0KLMea4TG
DzZSUD0pnhkGhFHtVbbtYjqMR66zR+AxKO65PofVLHInN61/KHiMUYL20dkgyLMn860
J5EWxJ3xLcO8pG1+dsOeechC1mKVMMRQ7MqcWufXHxfTn1Nm3lEfleMbz6H+tal5Feg
HozTTYIJQpoT/IkW/yAGb/m+fQqI2WqNkTb8hlBRbhZei731uiIntyXhMRaAL13zHD4
2yUTgpQvCb8A6NqUUpgyrAsQr3JbjMCKLSjcDO8fDFvUM6Z/cOE4SmcvwSlb0cQtrxN
3Jyi3rxR1X87n/aKlmk03KUxr5n8GzCim6bXPKly+ZFMHU3pC36kJr4oZgsWW5F3QUo
/5bqxWZ87iJuWmQY2JYsk2EFbRtHlet5LUq3mj92SRSd9sPuHu8c2NSyQ1Qym/3VRO2
KswFJppLShvO0Vwypj0bGL5P4CFtqibAkbIAz+QRv9qcsPhdQAZx9t0p+eFShJL8SEM
hMLL4aSCMmYaQw3OpkdBP4o/NBmQkRvL1xhCknH6pvtqcF0+R8pRX6w7rHlwT/j0gTs
QpUljlTeGnOCuXuxtio4W7+cT8uQrR+jF/7B9SsyLMvMUVxaPBeXAaDJY9mgLyY4BHc
e6gj/enRIVFcZtutj/86qwnsNm9/LJ3HKn4PcLjLUZQiuyc+OcdTflf9oP/QYrpvU2D
P+YrZ9czSE+U22DZHLprvp+JYJ62bbV/tBK9HgQrax5GkW/kwjZkODlRx2P2PoUA9Tp
nAsZIuh8xTVH4jOdkux1YeYudw==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
GhoqT0lJXnJLUmgoWyEKDuoLOLw9/lr2aBnObiY9j8NlB+sD7OA4k8lF7uUjgFv0oGX5RkftiQAjtBggXJ1VYKWrPvGTbrjKuD5VfQCYIE+CaVYrJqjyRE1mIiX5QXW7u+hgqgK46bt9ZlWedt3CKc9sM2l0Re7ThUbOq+SUYHBRZT85ffQG98MUeyKaS6FJG6edZFola78Ck146Xa0i5fE6RIrJBmSqin8xg0UD6ZDfT2SuBFjaPcr5WEiAqN2aH6ppAWu7ppMkD0OSBW3abmWnRvM/5BJFXsfsyQfsMXxX+HkejULcni8h2zg7mXo1NngLQiLR3eCbXh9wR9VpH5KQSWuMGcmj9YV6i5BRlczL3eIW564kySzTO6A5CLn7sELmeSu5YXE41LWNBh3cN9M2po+EyHWRfY7rV/rKLyjPa5W8sHQ/9IXxrfUDSLwkMItMRP0tKdFxPssyXpWhtvHXauFR63KYiDt83O109xJZv48ksVmQKLiD9ZH2yyaIjtzYHGUokfrCf9Hu/8aJwCfVCaTiysw0SGDHMu7IXW518s1eI6ecZKPXH0NlqUlaxQAXF1aCtFr0Vp9xn5TE6lqSZFI04g0t+sMTJZSlTGoG2N9V3rjtUi64tOi0WOEjNhPo/hcpFsiG5pOb9GuTk9taOPHBhljLPGon43Fv9tBmIXZFrvNfy3GXIeJ7Frn8gqTvqrwCA4K0TTff7qu/uiyZJFp/HxGuh1fHp5m1bSBiKhbujLv4V1aeg5AWI+GtMoRGkpzNFxXDwv40sICVbbY4iDDJLNx3cHXe6qwRb9Xl+jF+sa2fk4lQoMFtO4eYV6JAu5J07EcQ/VBDXg1SGSDJoEM8yw9TRyEOT3dFEbvLNFOhUfMMmgt75SbGGa8jhsjRQzHH1CjSUNr1gYZIJpaJjBPAr+MDmxIwwPpCj9qU4XfsYD+rFFP36MT+D63ShEHUdO1AhJlDTyoKBhSqPHmaoXaemvNpV4Dj4Iwr40YheLo532awAmp/MsbOODhcfeYSQk59DT+hr5+PMTYZGHqygo9BYT1tzpXnd5mRP/2jhmf4H5w3FescSUug5j6csm8F4sdl7Ik8PmtJZPXQN690ie0CPoZeps6TtYXCwthLnTG6lKK8o+bMAvvrcleAr/GG4b5hYfuLwCtWoZnFGFOnqtpLfG0KFjjvflD+t7RBXnQPkAxmgS2OkuzSbM9xPwiEAz/jFKBvLmsYvsDnKB1NtvlQDhYZ9kL3n3Pz6g5yHvoDV24wK6DxcAz2hq8JJX4fUm0Fp7iWG/BWFxVqBeHY6Om9klrP/FCB7jmp3Iw4wjBltVbz+evpxGfSDUhdwn8G0ECHVXxx3cJbbmlxng==
